<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-8PV5OBQH" ignored="0" name="LoginTest" tests="3" failures="2" timestamp="2021-11-24T18:52:08 IST" time="7.052" errors="0">
  <testcase name="runScenario" time="6.621" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="0.042" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at stepdefination.login.closure(login.java:62)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="0.01" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at stepdefination.login.closure(login.java:62)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- LoginTest -->
